frontend/src/router/index.ts

```diff
@@ -316,6 +316,40 @@ const routes: Array<RouteRecordRaw> = [
                 component: () => import("../views/SettingWorkspaceAgent.vue"),
                 props: true,
               },
+              {
+                path: "project",
+                name: "setting.workspace.project",
+                meta: {
+                  title: () => t("common.projects"),
+                  quickActionListByRole: () => {
+                    return new Map([
+                      [
+                        "OWNER",
+                        [
+                          "quickaction.bb.project.create",
+                          "quickaction.bb.project.default",
+                        ],
+                      ],
+                      [
+                        "DBA",
+                        [
+                          "quickaction.bb.project.create",
+                          "quickaction.bb.project.default",
+                        ],
+                      ],
+                      [
+                        "DEVELOPER",
+                        [
+                          "quickaction.bb.project.create",
+                          "quickaction.bb.project.default",
+                        ],
```

Comment:
Since we developers are not allowed to visit this route, how about returning empty array for developers here?

Reply:
Good catch. Done.